api/apps/api/test/jest-e2e.json

```diff
@@ -1,13 +1,7 @@
 {
-  "moduleFileExtensions": [
-    "js",
-    "json",
-    "ts"
-  ],
-  "moduleDirectories": [
-    "node_modules",
-    "src"
-  ],
+  "moduleFileExtensions": ["js", "json", "ts"],
+  "moduleDirectories": ["node_modules", "src"],
+  "testTimeout": 6000,
```

Comment:
Are we sure we use project's prettier settings?

Reply:
Fixed!

Reply:
(unless prettier didn't pick it up earlier for some reason)